cdk/pom.xml

```diff
@@ -11,11 +11,11 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdk.version>1.121.0</cdk.version>
     <junit.version>5.7.0</junit.version>
     <passay.version>1.6.0</passay.version>
-    <cdk-constructs.version>0.0.28</cdk-constructs.version>
+    <cdk-constructs.version>2.0.0-SNAPSHOT</cdk-constructs.version> <!-- TODO: Adjust with CDK v2 compatible version -->
```

Comment:
Waiting on the release of a AWS CDK v2 compatible release of `cdk-constructs` https://github.com/stratospheric-dev/cdk-constructs/pull/21